### ISSUE TITLE
set autopilot disengage inoperative

### ISSUE THREAD

**Issue by wbrinkman** [flybywiresim/a32nx#6131]
### Aircraft Version

Stable

### Build info

```json
{
  "built": "2021-10-28T18:01:48+00:00",
  "ref": "v0.7.1",
  "sha": "8d8fb7132f1be95a336f25a6e8441b1420d74989",
  "actor": "aguther",
  "event_name": "manual"
}
```

### Describe the bug

Activating the ingame bind for `set autpilot disengage` does not disenage the autopilot. The autopilot, either AP1 or AP2 remains active.

### Expected behavior

Pressing the keybind for `set autpilot disengage` should disengage all active autopilots, that is AP1 and AP2

### Steps to reproduce

1. Bind `set autpilot disengage` to a button
2. During an active flight, activate either AP1 or AP2
3. Press button set in step 1.
4. See that both AP1 and/or AP2 is still engaged (button is lighted on the glareshield, no audible alert)

### References (optional)

The bind for `toggle autopilot disengage` does deactivate an ongoing AP1/AP2, but this keybind when pressed again does re-engage the autopilot. 

Any logic that is used there might be copied over to fix the  `set autpilot disengage` keybind.

### Additional info (optional)

This was working before, however I've not played the game in a while, so unsure when this cropped up.

The stock Asobo a320 still handles this keybind correctly.

**Comment by aguther** [comment]
Well, a lot of events are popping up over time and we are not informed about them. Naming is also sometimes strange tbh.

You can also use Autopilot Off event which corresponds to the red button on the sidestick. This does what you want.

Here are also other events that can be used:
https://docs.flybywiresim.com/fbw-a32nx/feature-guides/autopilot-fbw/

I will check if the event can be captured and if so I will mal it to Autopilot Off function. Although it’s still unclear to me what the set in that means.

**Comment by wbrinkman** [comment]
I appreciate the difficulty in the many binds MSFS offers (and the abiguity in their naming), perhaps Asobo changed something internally that this is now a "press and hold" kind of thing. I do feel it is a bug as the behaviour is still in working condition for other aircraft, and the `set autopilot disengage` is not a new event, and was working before.

Thank you for suggesting the `autopilot off` instead, that does indeed covers the scenario I intend.

**Comment by aguther** [comment]
It was probably working because the stable version before 0.7 was using default Autopilot and therefore it was 1:1. Custom AP never implanted this event.

**Comment by aguther** [comment]
Is now also available on stable 0.7.2
